src/NamespaceTracker.hs

```diff
@@ -76,33 +77,36 @@ updateOwnerships i = mapM_ handle
             (modify (Map.delete $ namespace path))
     handle _ = return ()
 
+-- Takes an address with user and a list of owner messsage pairs
```

Comment:
I think if you comment like this (note `|` on first line) we can generate docs:
```haskell
-- | This is documentation, not just comments
-- Takes an address...
```
See https://www.haskell.org/haddock/doc/html/ch03s08.html

Reply:
I wasn't really intending this as docs, it was me trying to work out what the type meant. It should probably be removed if anything.

Reply:
Yeah, I thought that might be the case, just thought whilst it looked kinda docstring-y it might be worth sharing that haddock thing :-)